test_scripts/Defects/5_0/1909_SDL_must_set_LIMITED_HMILevel_in_case_app_was_activated_during_postponed_HMILevel_resumption.lua

```diff
@@ -0,0 +1,88 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/1909
+--
+-- Description:
+-- SDL must set LIMITED HMILevel in case app was activated during postponed HMILevel resumption
+-- Precondition:
+-- SDL and HMI are started.
+-- Media app is registered.
+-- In case:
+-- 1) Media app is resuming after unexpected disconnect.
+-- 2) HMILevel resumption of media app was postponed and SDL receives SDL.ActivateApp (<appID_of_media_app>).
+-- Expected result:
+-- 1) SDL must respond SDL.ActivateApp(SUCCESS) to HMI resume HMILevel of media app after event ends activate app and send OnHMIStatus (LIMITED, AUDIBLE) to this media app.
+-- Information: non-media apps:
+-- a. can be activated during active phone call
+-- b. can not be activated during OnEmergencyEvent (enabled=true) or active VR session (as there is no way to activate it during these events)
+-- Actual result:
+-- SDL does not set required HMILevel and audioStreamingState.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('user_modules/sequences/actions')
+local runner = require('user_modules/script_runner')
+local test = require("user_modules/dummy_connecttest")
+local utils = require("user_modules/utils")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+config.application1.registerAppInterfaceParams.appHMIType = { "MEDIA" }
+config.application1.registerAppInterfaceParams.isMediaApplication = true
+
+--[[ Local Functions ]]
+local function deactivateAppToLimited()
+    common.getHMIConnection():SendNotification("BasicCommunication.OnAppDeactivated", {appID = common.getHMIAppId()})
+    common.getMobileSession():ExpectNotification("OnHMIStatus",
+    {hmiLevel = "LIMITED", audioStreamingState = "AUDIBLE", systemContext = "MAIN"})
+end
+
+local function cleanSessions()
+    for i = 1, common.getAppsCount() do
```

Comment:
There is only one session, so session can be closed in ignitionOff be receiving BC.OnSDLClose